deps/socketio-0.1.0/src/socketio_handler.erl

```diff
@@ -71,7 +71,10 @@ handle(Req, HttpState = #http_state{action = create_session, config = #config{he
     HeartbeatTimeoutBin = list_to_binary(integer_to_list(HeartbeatTimeout div 1000)),
     SessionTimeoutBin = list_to_binary(integer_to_list(SessionTimeout div 1000)),
 
-    _Pid = socketio_session:create(Sid, SessionTimeout, Callback, Opts),
+    Pid = socketio_session:create(Sid, SessionTimeout, Callback, Opts),
+
+    % Adding this to be able to get client IP in bh_context
+    Pid ! {'req', Req},
```

Comment:
Ok, which `handle_info` clause handles this payload? And where is the code the do `Pid ! {'peer_ip', IP}`?